### DISCUSSION TITLE
"information_schema" doesn't show under "Databases" in navigation tree

### DISCUSSION THREAD

**Discussion by yonisade** [dbeaver/dbeaver#14281] [Q&A]
#### System information: 
- O/S: Windows 11
- DBeaver version 21.2.2.202110030714
- Java version 1.8.0_301

#### Connection specification:
- DB: MySQL 8.0.26 / 5.7
- Driver name: MySQL / MYSQL 5
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No

#### Describe the problem you're observing:
Cannot see "information_schema" under "Databases" in navigation tree

**Comment by LonwoLonwo** [comment]
Hello @yonisade 

Can you please check this setting?

![2021-10-18 11_22_14-Window](https://user-images.githubusercontent.com/45152336/137694907-a63669d5-ee7a-4dd1-8a5d-68a03be9f7a2.png)

Is it enabled?

**Reply by kuba-04** [reply]
works for me with H2, I landed here when looking for this table, thx
